recipes/tiny-optional/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import copy, get
+import os
+
+
+required_conan_version = ">=2.0.9"
```

Comment:
```suggestion
required_conan_version = ">=2.0"
```
v2.0.9 is only needed when `implements = ...` is used.